include/pybind11/pytypes.h

```diff
@@ -491,7 +491,7 @@ class tuple : public object {
         if (!m_ptr) pybind11_fail("Could not allocate tuple object!");
     }
     size_t size() const { return (size_t) PyTuple_Size(m_ptr); }
-    detail::tuple_accessor operator[](size_t index) const { return detail::tuple_accessor(ptr(), index); }
```

Comment:
Having it in the same PR is no problem -- I'm just wondering what the change from ``ptr()`` to ``*this`` is supposed to change here?